qdrant-landing/content/documentation/concepts/points.md

```diff
@@ -520,7 +520,7 @@ In this case, it means that points with the same id will be overwritten when re-
 Idempotence property is useful if you use, for example, a message queue that doesn't provide an exactly-ones guarantee.
 Even with such a system, Qdrant ensures data consistency.
 
-*Available as of v0.10.0*
+[*Available as of v0.10.0*](#create-vector-name)
```

Comment:
I set up an anchor here to support a reference from the collections.md file, line 683